airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml

```diff
@@ -5,34 +5,24 @@ tests:
   connection:
     - config_path: "secrets/config.json"
       status: "succeed"
-    - config_path: "secrets/connected_account_config.json"
```

Comment:
this should not be removed as it's testing an important feature in this connector